### ISSUE TITLE
Google Analytics 4 integration: download event registered for every bitstream download, regardless of bundle

### ISSUE THREAD

**Issue by bram-atmire** [DSpace/DSpace#8938]
**Describe the bug**
TL;DR GA4 download stats (= "item" events) are massively inflated.

Why: Bitstream downloads from the non-Original bundle, for example, thumbnail downloads, are also registered as "item" events in GA4 with action "download" and category "bitstream".

**To Reproduce**
1. Open up your realtime view in Google Analytics
2. Visit the homepage of your repository
3. Watch the item events roll in at a rate of 1 per item that is shown in recent submissions on the homepage

**Expected behavior**
Only bitstream downloads from the Original bundle should trigger a download event in Google Analytics.

**Related work**
Link to any related tickets or PRs here.
